src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -173,17 +173,18 @@ def get(self, request: Request, organization) -> Response:
         Select a field, define a date range, and group or filter by columns.
         """
         with self.handle_query_errors():
+            tenant_ids = {"organization_id": organization.id}
```

Comment:
does the referrer get injected somewhere else?

Reply:
Since referrer is passed to Snuba in every request, I've updated the Snuba query functions in `utils/snuba.py` to at least add the referrer per request into tenant ids.

Eg:

https://github.com/getsentry/sentry/blob/b92176660ed1103b97f3e7fa8625be9da23825da/src/sentry/utils/snuba.py#L719-L721